src/chains/optimism/vm/opcodes/block/coinbase.ts

```diff
@@ -0,0 +1,10 @@
+import { UndefinedOpcode } from '@/chains/types';
+
+export const coinbase: UndefinedOpcode = {
+  number: 0x41,
+  name: 'coinbase',
+  description: 'The opcode is not defined on Optimism',
```

Comment:
Let's also add a TODO here—from the optimism docs it's not clear to me if this means the opcode is a no-op (i.e. no stack inputs or outputs) or if it will revert